src/util.cpp

```diff
@@ -339,6 +339,10 @@ int LogPrintStr(const std::string &str)
     int ret = 0; // Returns total number of characters written
     static std::atomic_bool fStartedNewLine(true);
 
+    if (!fPrintToConsole && !fPrintToDebugLog) {
```

Comment:
nit: might be even more efficient to do this check in `util.h` already, before calling `tfm::format`.

Reply:
Good point, done.